src/ray/gcs/gcs_server.cc

```diff
@@ -75,17 +75,18 @@ GcsServer::GcsServer(
                            /*record_stats=*/true,
                            ClusterID::Nil(),
                            RayConfig::instance().gcs_server_rpc_client_thread_num()),
+      // NOTE: The raylet client server_unavailable_timeout_seconds is set to -1 because
+      // the gcs is notified when any node has died from the gcs node manager. Since we
+      // are on the gcs, we also don't need to deal with the dead node cache eviction race
+      // hence don't ever need to invoke the unavailable timeout callback.
```

Comment:
"dead node cache eviction race" is pretty much unintelligible to a future reader, they'd have to reverse engineer what you mean by this. state more plainly why we don't ever need rely on the callback.

Reply:
Where does the raylet client disconnect happen for GCS? Was that existing code?